### PR TITLE
[MO] How to analyse and react to a bug on a React-Native project

### PR DESCRIPTION
**Pull request by tpucci**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/tpucci/bug/react-native/debugging/analyse-bug.mo.md">analyse-bug.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>


## Automatic reviewers
  
cc: @louiszawadzki, @tychota
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/debugging/analyse-bug.mo.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+{% hint style='info' %}
+
+For M33 People:
+
+First thing first: communicate with your Product Owner and follow this [mode of operation](http://blog.m33.network/2017/07/i-just-detected-a-bug-what-should-i-do/).
+
+{% endhint %}
+
+### Option 1: You know where it comes from and how to fix it
```
Comment (louiszawadzki) [inline react-native/debugging/analyse-bug.mo.md]: You never know from first sight where it comes from, how much spend time should you spend before switching to option 2?

File: react-native/debugging/analyse-bug.mo.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+Ask for the person who spotted the bug all information of its environment. Such as:
+
+* app version;
+* account information (username, password, mean of payment);
+* exact mode of operation which causes the bug (list of actions and waiting times): this can includes:
+  * time and dates
+  * localisation
+  * phone orientation
+  * connectivity information
```
Comment (louiszawadzki) [inline react-native/debugging/analyse-bug.mo.md:54]: Can you give a good and a bad example?

Comment (louiszawadzki) [reply react-native/debugging/analyse-bug.mo.md:54]: If the user is logged in, which data is on the account, and whether you can use this account or not -> example on a banking project, only the PO's real account had some data that wasn't in our mock data

**Pull request merged**